corehq/apps/reports/tests/sql_reports.py

```diff
@@ -21,6 +23,7 @@ def combine_indicator(num, denom):
         return None
 
 
+@nottest
 def test_report(report, keys=None, filters=None, group_by=None):
```

Comment:
I agree, that seems like a better name, but it would not change the need for the `@nottest` decorator. See the [extended usage](https://nose.readthedocs.org/en/latest/usage.html?highlight=ignore#extended-usage) section of the nose docs for a description of what things are discovered as tests.